pkg/apiserver/apiserver.go

```diff
@@ -319,22 +319,29 @@ func installHandlers(c *ExtraConfig, s *genericapiserver.GenericAPIServer) {
 func DefaultCAConfig() *certificate.CAConfig {
 	return &certificate.CAConfig{
 		CAConfigMapName: certificate.AntreaCAConfigMapName,
-		APIServiceNames: []string{
-			"v1alpha1.stats.antrea.io",
-			"v1beta1.system.antrea.io",
-			"v1beta2.controlplane.antrea.io",
+		APIServiceSelector: &metav1.LabelSelector{
+			MatchLabels: map[string]string{
+				"app":       "antrea",
+				"served-by": "antrea-controller",
+			},
```

Comment:
You can define a variable for this before line 320.

Reply:
done.